script/coocook_docker.pl

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env perl
+
+# ABSTRACT: helper script to control the coocook instance inside a
+# Dockerimage from the host OS
```

Comment:
needs to be 1 line for some applications to make use of this

Reply:
Thanks, I didn't know this. I merged the lines.